Mono.Cecil/AssemblyWriter.cs

```diff
@@ -3090,7 +3090,7 @@ static string GetXmlSecurityDeclaration (SecurityDeclaration declaration)
 
 		void WriteTypeReference (TypeReference type)
 		{
-			WriteUTF8String (TypeParser.ToParseable (type));
+			WriteUTF8String (TypeParser.ToParseable (type, false));
```

Comment:
Mentionned in the PR (sorry the commit message wasn't very explicit as to why):

> Quick note, the other patch is to make fixed array work with an assembly that roundtrips through Cecil.

Basically if mscorlib full name is not stored in the `FixedBufferAttribute` used for stuff such as `public fixed int[4]`, the library exposing this can't be consumed (both csc compiler and intellisense complains).